lib/board.rb

```diff
@@ -0,0 +1,71 @@
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+class Board
+  attr_accessor :board
+  attr_reader :inputs
+
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @inputs = 0
+  end
+
+  def display_board
+    puts "#{@board[0]} | #{@board[1]} | #{@board[2]}"
+    puts '--|---|--'
+    puts "#{@board[3]} | #{@board[4]} | #{@board[5]}"
+    puts '--|---|--'
+    puts "#{@board[6]} | #{@board[7]} | #{@board[8]}"
```

Comment:
There should be no puts, print, or get.chomp outside of the main.rb file. 
![only_main_print](https://user-images.githubusercontent.com/32361963/94994901-462b8f00-05a3-11eb-9bf0-322e0e75073c.png)